src/usePopper.js

```diff
@@ -29,28 +71,58 @@ export default function usePopper(
   {
     enabled = true,
     placement = 'bottom',
-    positionFixed = false,
+    strategy = 'absolute',
     eventsEnabled = true,
-    modifiers = {},
+    ...options
```

Comment:
what's the point of spreading out `...options` if you ignore all the ones other than `modifiers`?